content/vanus-ai/how-to/discord.md

```diff
@@ -0,0 +1,54 @@
+# Discord Integration
+This documentation will teach you how to integrate an AI into Discord.
+Integration is a paid user feature. Therefore, only users with subscriptions or 14 days trials can use this feature.
+
+## how to Setup a Discord Integration
+
+### Step 1: Create an AI Application
+1. Go to [ai.vanus.ai](https://ai.vanus.ai) and sign in.
+2. Create an new AI app by clicking **Create AI**.
+![](images/discord-createapp.png)
+3. Enter a name, the model, upload to the knowledge base, and click **Create**.
```

Comment:
Enter a name, the *app* model, upload to the knowledge base, and click **Create**.